azure-pipelines.yml

```diff
@@ -0,0 +1,59 @@
+jobs:
+  - job:
+    timeoutInMinutes: 120
+    strategy:
+      matrix:
+        Mojave-Xcode-10.2.1:
+          IMAGE_POOL: 'macOS-10.14'
+          XCODE_VERSION: '10.3'
+        Mojave-Xcode-11.0:
+          IMAGE_POOL: 'macOS-10.14'
+          XCODE_VERSION: '11'
+    pool:
+      vmImage: $(IMAGE_POOL)
+
+    steps:
+
+    - script: |
+        set -e
+        sudo xcode-select --switch "/Applications/Xcode_$(XCODE_VERSION).app/Contents/Developer"
+        echo "Xcode version: $(xcrun xcodebuild -version)"
+        echo "Xcode path: $(xcrun -f xcodebuild)"
+        echo "Xcode select path: $(xcode-select --print-path)"
+      displayName: "Select Xcode $(XCODE_VERSION)"
+
+    - script: git submodule update --init --recursive
+      displayName: "Update submodules"
+
+    - script: |
+        set -e
+        gem update --system
+        gem uninstall -Vax --force --no-abort-on-dependent calabash-cucumber
```

Comment:
I think we don't need to do that because ADO VM is empty by default

Reply:
Done. I've left only updating of gems